pkg/apis/marketplace/v1alpha1/operatorsource_types.go

```diff
@@ -13,19 +13,6 @@ const (
 	OpSrcFinalizer = "finalizer.operatorsources.marketplace.redhat.com"
 )
 
-// +genclient
-// +genclient:noStatus
-// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
-
-// OperatorSource is the Schema for the operatorsources API
-// +k8s:openapi-gen=true
-type OperatorSource struct {
-	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata,omitempty"`
-	Spec              OperatorSourceSpec   `json:"spec,omitempty"`
-	Status            OperatorSourceStatus `json:"status,omitempty"`
-}
-
```

Comment:
Same question: did you have to move it to get `operatorsdk k8s generate` to work? I don't see the `// +genclient` tag that was added.

Reply:
@tkashem I didn't use the sdk to generate it, I just pulled your commit down. I can undo this and reapply it using the sdk if needed.